src/h5web/visualizer/Visualizer.tsx

```diff
@@ -22,31 +22,33 @@ function Visualizer<T extends VisDef>(props: Props<T>) {
     return <p className={styles.fallback}>Nothing to visualize</p>;
   }
 
-  const { Container, Toolbar } = activeVis;
+  const { Container, Toolbar, ConfigProvider = Fragment } = activeVis;
 
   return (
     <div className={styles.visualizer}>
-      <div className={styles.visBar}>
-        <VisSelector
-          activeVis={activeVis}
-          choices={supportedVis}
-          onChange={onActiveVisChange}
-        />
-        {Toolbar && <Toolbar />}
-      </div>
+      <ConfigProvider>
```

Comment:
I render the active visualization's config provider here. Every time the active vis changes, the provider is unmounted and then another one is mounted, which means the `useState` of the latter invokes its lazy initializer to recreate the config store.